Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -38,66 +37,64 @@ export class AppComponent implements OnInit, OnDestroy {
   ) {}
   public ngOnInit() {
     this.store.dispatch(new InitTheme());
-    // Commented as long as serviceWorker is reinstalled
-    // Issue: https://github.com/T-Systems-MMS/phonebook/issues/87
-    // //Checking if the Service Worker was installed correctly.
-    // if (!this.store.selectSnapshot(AppState.serviceWorkerNotificationDisplayed)) {
-    //   if ('serviceWorker' in navigator) {
-    //     this.swUpdates.activated.subscribe(active => {
-    //       if (active.current) {
-    //         this.snackBar.open(
-    //           this.i18n({
-    //             value: 'Hurray! You can use this Website offline.',
-    //             description: 'Message indicating the service worker was successfully installed',
-    //             id: 'AppComponentServiceWorkerSuccessMessage',
-    //             meaning: 'AppComponent'
-    //           }),
-    //           '',
-    //           { duration: 3000 }
-    //         );
-    //         this.store.dispatch(new ServiceWorkerNotificationDisplayed());
-    //       }
-    //     });
-    //   } else {
-    //     this.snackBar.open(
-    //       this.i18n({
-    //         value: 'Your Browser does not support Offline Apps!',
-    //         description: 'Message indicating the service worker was not installed',
-    //         id: 'AppComponentServiceWorkerErrorMessage',
-    //         meaning: 'AppComponent'
-    //       }),
-    //       '',
-    //       {
-    //         duration: 3000
-    //       }
-    //     );
-    //     this.store.dispatch(new ServiceWorkerNotificationDisplayed());
-    //   }
-    // }
+    //Checking if the Service Worker was installed correctly.
+    if (!this.store.selectSnapshot(AppState.serviceWorkerNotificationDisplayed)) {
+      if ('serviceWorker' in navigator) {
+        this.swUpdates.activated.subscribe(active => {
+          if (active.current) {
```

Comment:
When should that be opened? Didn't got any message.

Reply:
okay. Got it.

Reply:
should have opened! I will investigate. Good catch

Reply:
I got one dialog "Update your site" or something like..